swarm/storage/mru/resource.go

```diff
@@ -134,7 +148,8 @@ func (r *resource) NameHash() common.Hash {
 	return r.nameHash
 }
 
-func (r *resource) Size(chan bool) (int64, error) {
+// implements (which?) interface
```

Comment:
`storage.LazySectionReader`?

Reply:
Confirmed it is `storage.LazySectionReader`. Implemented so a resource can be returned from an `API.Get()` call. Fixed comment in #704